src/protocol.h

```diff
@@ -29,6 +27,16 @@
 class CMessageHeader
 {
 public:
+    enum {
+        MESSAGE_START_SIZE = 4,
```

Comment:
nit: could make these constexprs instead while we're at it. No real reason other than cosmetic, though.